Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -81,7 +81,7 @@ open class ChatMessageListVC: _ViewController,
     open var isScrollToBottomButtonVisible: Bool {
         let isMoreContentThanOnePage = listView.contentSize.height > listView.bounds.height
 
-        return !listView.isLastCellFullyVisible && isMoreContentThanOnePage
+        return !listView.isLastCellFullyVisible && isMoreContentThanOnePage || dataSource?.isFirstPageLoaded == false
```

Comment:
I assume that the following precedence is what you wanted
```suggestion
        return (!listView.isLastCellFullyVisible && isMoreContentThanOnePage) || dataSource?.isFirstPageLoaded == false
```

It will be great to add parenthesis to help other devs easily and quickly understand the precedence here.